momentumopt/python/momentumopt/motion_planner.py

```diff
@@ -182,34 +162,82 @@ def replay_kinematics(self, start=0, end=None, viz=None):
             viz.display(np.matrix(q).T)
             time.sleep(self.kin_optimizer.dt)
 
-    def plot_base_trajecory(self, start=0, end=None):
+    def replay_kinematics_meshcat(self, start=0, end=None):
```

Comment:
I wonder if we should just change the default viewer to `meshcat` which is called from a `replay_kinematics` method and add a new `replay_kinematics_gepetto` in case someone wants to use the older viewer?

Reply:
or you create 3 methods:
- replay_kinematics_meshcat
- replay_kinematics_gepetto
- replay_kinematics(viewer="meshcat")